code/game/machinery/kitchen/gibber.dm

```diff
@@ -235,15 +235,22 @@ obj/machinery/gibber/New()
 	src.occupant = null
 
 	spawn(src.gibtime)
+		var/no_more_gibs = FALSE
 		operating = 0
 		for (var/i=1 to totalslabs)
 			var/obj/item/meatslab = allmeat[i]
 			var/turf/Tx = locate(src.x - i, src.y, src.z)
 			meatslab.forceMove(src.loc)
 			meatslab.throw_at(Tx,i,3)
 			if (!Tx.density)
-				var/obj/effect/decal/cleanable/blood/gibs/O = getFromPool(/obj/effect/decal/cleanable/blood/gibs, Tx)
-				O.New(Tx,i)
+				if(!no_more_gibs)
+					var/obj/effect/decal/cleanable/blood/gibs/O = getFromPool(/obj/effect/decal/cleanable/blood/gibs, Tx)
```

Comment:
After having looked at the getFromPool proc, everything after the first argument is passed to the first arguments New, so calling O.New is pointless.

Change it to getFromPool(/obj/effect/decal/cleanable/blood/gibs, Tx, i) and remove the O.new(...), same with the one below (Except have it be getFromPool(/obj/effect/decal/cleanable/blood/gibs, locate(src), i) instead)

Reply:
Not `locate(src)` since that doesn't make sense